docs/source/cloud-setup/cloud-permissions/aws.rst

```diff
@@ -4,36 +4,195 @@
 AWS
 =====
 
+.. _aws-sso:
+
+Using AWS SSO
+-------------
+
+`AWS IAM Identity Center <https://aws.amazon.com/iam/identity-center/>`_ (Successor to AWS Single Sign-On, or SSO) is supported.
+
+.. warning::
+  SSO login *will not work across multiple clouds*. If you use multiple clouds, you should :ref:`set up a dedicated user and credentials <dedicated-aws-user>` so that instances launched on other clouds can use AWS resources.
+
+To use it, ensure that your machine `has AWS CLI V2 installed <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`_ (by default, ``pip install skypilot[aws]`` installs V1; V2 cannot be installed via pip).
+
+You can use the following to check version:
+
+.. code-block:: console
+
+    $ aws --version
+    aws-cli/2.25.6 ...
+
+Visit your SSO login portal (e.g. ``https://my-sso-portal.awsapps.com/start``), and click on **Access keys** under the corresponding account. Under "AWS IAM Identity Center credentials (Recommended)" note:
+
+- the ``SSO start URL``
+- the ``SSO Region``
+
+Then, log into your SSO account:
+
+.. code-block:: console
+
+    $ aws configure sso
+
+- **SSO session name**: should be set, but you can choose any name you want.
+- **SSO start URL**: copy from the SSO login portal
+- **SSO region**: copy from the SSO login portal
+- **SSO registration scopes**: leave blank to use the default
+
+Log in and approve the request in your web browser. Then back in the CLI, complete the remaining fields:
+
+- **Default client Region**: optional
+- **CLI default output format**: optional
+- **Profile name**: set to ``default`` if you want to use this profile by default with SkyPilot, otherwise see :ref:`several-aws-profiles`.
+
+If everything is set up correctly, :code:`sky check aws` should succeed!
```

Comment:
We need to emphasize the requirement of setting `AWS_PROFILE` in the env var. Otherwise, it won't work for SkyPilot.

Reply:
If you follow the instructions to set profile name to `default` (line 46), it's not needed to set AWS_PROFILE. This part is already complicated so I don't want to add too much detail here. I'm adding a troubleshooting section that will include checking AWS_PROFILE.